pages/linux/kdialog.md

```diff
@@ -0,0 +1,36 @@
+# kdialog
+
+> Show KDE dialog boxes from within shell scripts.
+> More information: <https://develop.kde.org/deploy/kdialog/>.
+
+- Open a dialog box displaying a custom message:
+
+`kdialog --msgbox "{{message}}" "{{optional_detailed_message}}"`
+
+- Open a question dialog with a `yes` and `no` button, returning `0` and `1`, respectively:
+
+`kdialog --yesno "{{message}}"`
+
+- Open a warning dialog with a `yes`, `no`, and `cancel` button, returning `0`, `1`, or `2` respectively:
+
+`kdialog --warningyesnocancel "{{message}}"`
+
+- Open an input dialog box and print the input to stdout when `OK` is pressed:
+
+`kdialog --inputbox "{{message}}" "{{optional_default_text}}"`
+
+- Open a dialog to prompt for a password and print it to stdout:
+
+`kdialog --password "{{message}}"`
+
+- Open a dialog containing a dropdown menu and print the selected item to stdout:
```

Comment:
```suggestion
- Open a dialog containing a specific dropdown menu and print the selected item to stdout:
```

Reply:
How about `predefined` instead?

Reply:
Although it's a good idea but to make all pages more standardized it's better to stick to smth one. 😄 For instance to `specific`.